spec/components/credit_card_spec.rb

```diff
@@ -87,5 +87,11 @@
         was_3d_secure_successful: true,
       )
     end
+
+    it 'accepts country code' do
```

Comment:
suggestion: for the sake of consistency with the other tests, the descriptions could be something along the lines of "does not raise an exception  when country code is given but issuer_id_number" is not provided.